DESCRIPTION

```diff
@@ -29,7 +29,7 @@ Imports:
 LinkingTo: StanHeaders (>= 2.18.1), rstan (>= 2.18.1), BH (>= 1.66.0-1), Rcpp (>= 0.12.17), RcppEigen (>= 0.3.3.4.0)
 SystemRequirements: GNU make
 NeedsCompilation: yes
-RoxygenNote: 6.1.1
+RoxygenNote: 7.0.2
```

Comment:
Minor, but please remove this and also delete .Rd files. I want to update Roxygen on my end and update .Rd files after we merge. Otherwise there will be stupid clashes when merging to devel

Reply:
Alright--I found that if I wanted linking within the documentation to work I needed to update roxygen2. I can delete the note and the .rd files though.